### PR TITLE
docs: Npm Packages guide for CLI

### PR DESCRIPTION
**Pull request by wardbell**
Issue #19510 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: aio/content/guide/npm-packages.md [hunk 1]
```diff
@@ -20,244 +22,152 @@ Consider using [nvm](https://github.com/creationix/nvm) for managing multiple
...
-
-### Other helper libraries
-
-***angular-in-memory-web-api***: An Angular-supported library that simulates a remote server's web api
-without requiring an actual server or real HTTP calls.
-Good for demos, samples, and early stage development (before you even have a server).
-
-***bootstrap***: [Bootstrap](http://getbootstrap.com/) is a popular HTML and CSS framework for designing responsive web apps.
-Some of the samples improve their appearance with *bootstrap*.
+**[zone.js](https://github.com/angular/zone.js)**: Angular relies on zone.js to run Angular's change detection processes when native JavaScript operations raise events.  Zone.js is an implementation of a [specification](https://gist.github.com/mhevery/63fdcdf7c65886051d55) currently before the
```
Comment (Toxicable) [inline aio/content/guide/npm-packages.md]: I might be being a bit pedantic here but Angular dosen't "rely" on zones to handle CD, it uses it by default but that's optional.
Maybe reword it in a way to reflect this point?

**Review by Foxandxss — APPROVED** [review]

**Comment by vicb** [comment]
missing:
- target label
- merge label
